crates/daphne/src/vdaf/prio2.rs

```diff
@@ -133,7 +134,7 @@ pub(crate) fn prio2_prep_finish(
             )))
         }
     };
-    let agg_share = VdafAggregateShare::Field32Draft09(vdaf.aggregate(&(), [out_share])?);
```

Comment:
We should be able to remove the `Field32Draft09` variant now.

Reply:
Doesn't `Pine32HmacSha256Aes128` depend on it?